### DISCUSSION TITLE
Able to organize/name personal vehicles

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#2086] [Ideas]
is it possible or is it fine to provide a way to organize or give custom names to personal vehicles. This will help them retrieve quickly.

![image](https://github.com/YimMenu/YimMenu/assets/139460769/a3bb5903-bc6d-49bf-82e8-5be5b3e64948)

Current solution is to save your personal vehicle as persistent vehicle with custom name or in custom folder (new).

**Comment by pierrelasse** [comment]
It would be cool if you could also open the mecanics menu instead of this so you can see in what garage the vehicle is and it would be cool if you could preview the vehicle as a picture (saw this in some yt vid) instead of spawning it in the world where it is hard to see.

**Reply by Rabtos** [reply]
Yeah garage list would be great. Maybe an Sub Menu above "Vehicle Class" and "Model Name" where a list of your garages is shown.
I like the preview of the vehicle in 3D. So I can see all sides of it. I don't think its soo hard to see.

**Reply by Yimura** [reply]
Since we're hooking the function responsible for all the labels in the game we could abuse this to rename garages in-game to our own liking.
For example Maze Bank we could name Muscle Garage if all our muscle cars are in there.

**Comment by lonelybud** [comment]
We can close this discussion now. Thanks @gir489returns for the effort.
